src/_locales/zh/messages.json

```diff
@@ -347,5 +347,26 @@
     "welcome_1b_dragging": "將備份檔案拖拽到此處……",
     "welcome_1b_file_selected": "已選取備份檔案：{{filename}}",
     "untitled_wallet": "未命名錢包",
-    "your_message": "您的留言……"
+    "your_message": "您的留言……",
+    "plugin_file_service_display_name": "插件: 文檔服務",
```

Comment:
```suggestion
    "plugin_file_service_display_name": "文檔服務",
```

Reply:
@neruthes

Reply:
i mean it should not necessary for evey plugin includes "Plugin:" prefix in its name.

Reply:
The name of each plugin does not have to begin with "Plugin: ". However, notice that the title of the Plugin Screen Modal (PSM) should be "Plugin: ${plugin_name_with_i18n}".

Reply:
https://github.com/DimensionDev/Maskbook/blob/827f697a594c182f0a3b8e49f3deac1ac7cccc86/src/plugins/Wallet/UI/RedPacket/RedPacketDialog.tsx#L420-L422

why?